btr-api/README.md

```diff
@@ -1,4 +1,4 @@
-[![License](https://img.shields.io/badge/License-BSD%203%20Clause-blue.svg)](LICENSE)
+[![License](https://img.shields.io/badge/License-BSD%203%20Clause-blue.svg)](../LICENSE)
```

Comment:
was this on purpose?

Reply:
Yes, Thor suggested that we leave license in the root dir.